### ISSUE TITLE
G2H Pro - working or not ???

### ISSUE THREAD

**Issue by calaba** [mcchas/g2h-camera-mods#45]
Hi all,

did anyone get tis SD card hostname hack working on G2H Pro? Which firmware version? 

Mine FW 3.4.6 seems not to work. No time to play with attaching the serial console for debugging :( ... 

Did anyone dumped whole file system to tgz of some FW where the hack worked ?? 

Can anyone / author post the serial console boot log here ???

Any other ways how to get root / RTSP working on G2H Pro ?

Thanx for sharing any experience.

**Comment by wizardofozzie** [comment]
> Hi all,
> 
> did anyone get tis SD card hostname hack working on G2H Pro? Which firmware version?
> 
> Mine FW 3.4.6 seems not to work. No time to play with attaching the serial console for debugging :( ...
> 
> Did anyone dumped whole file system to tgz of some FW where the hack worked ??
> 
> Can anyone / author post the serial console boot log here ???
> 
> Any other ways how to get root / RTSP working on G2H Pro ?
> 
> Thanx for sharing any experience.

Yes, with help from @niceboygithub I did. Msg back if you need an update

**Comment by niceboygithub** [comment]
Did you see the guide?


SDcard hostname method only works on G2H, not "G2H Pro".

To hack G2H Pro, use aQRootG3 on old firmware. (Thanks for Wh1terat)

**Comment by wizardofozzie** [comment]
> Did you see the guide?
> 
> SDcard hostname method only works on G2H, not "G2H Pro".
> 

I did but was confused which camera I owned.
@niceboygithub thanks for doing all this.
Did you make the 3.3.4 firmware on your own? How?
